locales/en.yml

```diff
@@ -0,0 +1,28 @@
+en:
+  theme_metadata:
+    custom_header_links: "Comma delimited in this order: link text, link title, URL, view, target, hide on scroll<br><b>Link text:</b> The text for the link<br><b>Link title:</b> the text that shows when the link is hovered<br><b>URL:</b> The path for the link (can be relative)<br><b>View:</b> vdm = desktop and mobile, vdo = desktop only, vmo = mobile only<br><b>Target:</b> blank = opens in a new tab, self = opens in the same tab<br><b>Hide on scroll:</b> remove = hides the link when the title is expanded on topic pages keep = keeps the link visible even when the title is visible on topic pages<br><b>Language:</b> blank = no locale assoaciated to the link, else insert a locale code (en, fr, de, ...)"
+    settings:
+      Header_links:
+        description: Custom links to be displayed in the header
+        schema:
+          properties:
+            title:
+              label: Title
+              description: The title attribute for the link
+            icon:
+              label: Icon
+              description: The icon used for the link
+            url:
+              label: URL
+              description: The URL for the link
+            view:
+              label: View
+              description: |
+                vdm = desktop and mobile
+                vdo = desktop only
+                vmo = mobile only
+            target:
+              label: Target
+              description: |
+                blank = opens in a new tab
+                self = opens in the same tab
```

Comment:
Do we have support for enums here? Would be great if we can present them as dropdowns rather than free text fields